### DISCUSSION TITLE
What does "Alt Speeds" function do?

### DISCUSSION THREAD

**Discussion by marcusique** [VueTorrent/VueTorrent#706] [Q&A] [answered]
I thought you could change global/alternative speed limits, but I couldn't find this option in settings (I believe it exists in the native UI), so that Alt Speeds function toggle between the two. The only thing I noticed is that it reduces the download speed of all my torrents to 2-3 KB/s, but I can't find any way to change that. What is this function used for?
![Screenshot 2023-03-13 at 15 10 12](https://user-images.githubusercontent.com/23212368/224727168-3506d4cc-9e7e-4b3e-8601-5803ee44f665.png)

**Comment by Larsluph** [comment, marked as answer]
Try clicking on the speedometers in the navbar (on top of the speed graph) to change speed limits

**Reply by Larsluph** [reply]
You can also configure those limits in qBittorrent or in the "settings" page under the "Speed" tab.
